Build failure in CoreTextCompositionEngine.h when the build is not unified

graouts opened the pull request:
#### d838315bb27b4b285f2c883b0b94432830b383f1
<pre>
Build failure in CoreTextCompositionEngine.h when the build is not unified
<a href="https://bugs.webkit.org/show_bug.cgi?id=253017">https://bugs.webkit.org/show_bug.cgi?id=253017</a>

Reviewed by Alexey Proskuryakov.

* Source/WebCore/platform/text/mac/CoreTextCompositionEngine.h:

Canonical link: <a href="https://commits.webkit.org/260925@main">https://commits.webkit.org/260925@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/78277447ddb1defa391a8db460eb0a8b86341194

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/109872 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/18982 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/42584 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/1329 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118955 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/113822 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/20448 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/10177 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/102141 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/115620 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/15224 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/98444 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/43446 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/97187 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/30100 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/85251 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/11692 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/31442 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/12330 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/8390 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/17698 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/51051 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/14106 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/4109 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov commented on Source/WebCore/platform/text/mac/CoreTextCompositionEngine.h:
```diff
@@ -25,6 +25,8 @@
 
+#include <pal/spi/cf/CoreTextSPI.h>
+
 using CharacterClass = uint8_t;
```
Also, this seems unnecessary?

graouts committed:
Build failure in CoreTextCompositionEngine.h when the build is not unified
https://bugs.webkit.org/show_bug.cgi?id=253017

Reviewed by Alexey Proskuryakov.

* Source/WebCore/platform/text/mac/CoreTextCompositionEngine.h:

Canonical link: https://commits.webkit.org/260925@main
